OasisDEX/oasis-borrow
# Update vault history section

# [Update vault history section](https://app.shortcut.com/oazo-apps/story/3860/redesign-update-vault-history-section)
  
## Changes 👷‍♀️

- Updated layout in vault history section.
- Removed extensive usage of `div`s all over the place and changed some of them into other tags according to `HTML` semantics.
  
## How to test 🧪

Check out manage vault page or, in case of automation feature flag enabled, "history" tab on vault page. Please not that you need to check multiply vault to see events on in history that has details.
